www/includes/easyparliament/templates/html/static/privacy.php

```diff
@@ -39,50 +163,13 @@
-
-            <p>In technical speak, this allows Google to collect data about your
-            traffic via Google
-            <a href="http://www.google.com/policies/technologies/types/">advertising
-            cookies</a> and
-            <a href="https://www.google.com/policies/privacy/key-terms/#toc-terms-identifier">anonymous
-            identifiers</a>, in addition to data collected through a standard
-            Google Analytics implementation. Regardless of the source of the
-            data, we strictly adhere to
-            <a href="https://support.google.com/analytics/answer/2700409">Google&rsquo;s
-            policy requirements</a> in our treatment of your data. We do not
-            facilitate the merging of personally-identifiable information with
-            non-personally identifiable information collected through any Google
-            advertising product or feature.</p>
-
-            <h3>Google&rsquo;s Official Statement about Analytics Data</h3>
-            <p>&ldquo;This website uses Google Analytics, a web analytics
-            service provided by Google, Inc. (&ldquo;Google&rdquo;). Google
-            Analytics uses &ldquo;cookies&rdquo;, which are text files placed on
-            your computer, to help the website analyze how users use the site.
-            The information generated by the cookie about your use of the
-            website (including your IP address) will be transmitted to and
-            stored by Google on servers in the United States. Google will use
-            this information for the purpose of evaluating your use of the
-            website, compiling reports on website activity for website operators
-            and providing other services relating to website activity and
-            internet usage. Google may also transfer this information to third
-            parties where required to do so by law, or where such third parties
-            process the information on Google&rsquo;s behalf. Google will not
-            associate your IP address with any other data held by Google. You
-            may refuse the use of cookies by selecting the appropriate settings
-            on your browser, however please note that if you do this you may not
-            be able to use the full functionality of this website. By using this
-            website, you consent to the processing of data about you by Google
-            in the manner and for the purposes set out above.&rdquo;</p>
-
-            <h3>Opting out</h3>
+            and information about your web browser. IP addresses are masked
+            (only a portion is stored) and personal information is only reported
+            in aggregate. We do not allow Google to use or share our analytics
+            data for any purpose besides providing us with analytics information,
+            and we recommend that any user of Google Analytics does the same.</p>
+
+            <h3>Opting out of Google Analytics tracking</h3>
```

Comment:
This section text doesn't match the doc.

Reply:
I’ve changed the heading from "Opting out" (in the document) to "Opting out of Google Analytics tracking" to make it clearer what you’re opting out of. The alternative was to make this heading a `<h4>`, but annoyingly I’d also then need to fix the CSS styles to make `<h4>`s appear smaller than `<h3>` (sigh) and just amending the heading text seemed simpler.

Reply:
I meant the section (body) text, not the heading text, that's why I said this section text, sorry wasn't clear enough.

Reply:
But you have changed that so all good.

Reply:
Yes, sorry, I knew what you meant. Was just leaving a comment as to why I _hadn’t_ changed the heading text. All good now anyway.